macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -689,11 +689,10 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
 
         options[NetworkProtectionOptionKey.excludeLocalNetworks] = NSNumber(value: settings.excludeLocalNetworks)
 
-#if NETP_SYSTEM_EXTENSION
```

Comment:
This was the issue.  This target had `NETP_SYSTEM_EXTENSION` removed.  It really doesn't matter if we encode the data below for all targets.